source/delegate/contracts/interfaces/IDelegate.sol

```diff
@@ -42,6 +42,15 @@ interface IDelegate {
     address indexed signerToken
   );
 
+  event ProvideOrder(
+    address indexed ruleOwner,
```

Comment:
`delegateOwner`?

Reply:
its called `delegateOwner` in the other events currently. (setrule and unsetrule) but could rename it in those too

Reply:
I see. Maybe similar to `staker` elsewhere this could just be `owner` in both cases.

Reply:
Did you think it was worth having both owner + tradeWallet but indexing by owner?